lib/relationships.js

```diff
@@ -0,0 +1,96 @@
+var _ = require('lodash');
+var inflection = require('inflection');
+
+module.exports = function (app, options) {
+  //get remote methods.
+  //set strong-remoting for more information
+  //https://github.com/strongloop/strong-remoting
+  var remotes = app.remotes();
+
+  remotes.before('**', function (ctx, next) {
+
+    var id = ctx.req.params.id;
+    var data = ctx.args.data;
+
+    relationships(id, data, app);
+
+    next();
+  });
+
+  // for create
+  remotes.after('**', function (ctx, next) {
+
+    if (!ctx.result) {
+      next();
+    };
+    var id = ctx.result.id;
+    var data = ctx.req.body;
+
+    relationships(id, data, app);
+
+    next();
+  });
+
+};
+
+function relationships (id, data, app) {
+
+  if (!data || !data.data || !data.data.type || !id) {
+    return;
+  }
+
+  var modelName = inflection.classify(data.data.type);
```

Comment:
I'm wondering if its possible to avoid the use of the inflection module since its possible that someone could define in their model.json file a model named for example `cat` with a plural form `dogs` (makes no sense I know but its possible) in which case inflection.classify would get it wrong. 

You could probably get it from:
- data.relationships via modelFrom or modelTo (would require a little work though since you would have to check if its a belongsTo or hasMany to know whether to use modelFrom or modelTo.
- its probably available on ctx as something like ctx.method.sharedClass